sounddevice.py

```diff
@@ -2776,20 +2803,27 @@ def _split(value):
     return invalue, outvalue
 
 
-def _check(err, msg=""):
+def _check(err, msg=None):
     """Raise error for non-zero error codes."""
-    if err < 0:
-        msg += ': ' if msg else ''
-        if err == _lib.paUnanticipatedHostError:
-            info = _lib.Pa_GetLastHostErrorInfo()
-            hostapi = _lib.Pa_HostApiTypeIdToHostApiIndex(info.hostApiType)
-            msg += 'Unanticipated host API {0} error {1}: {2!r}'.format(
-                hostapi, info.errorCode, _ffi.string(info.errorText).decode())
-        else:
-            msg += _ffi.string(_lib.Pa_GetErrorText(err)).decode()
-        raise PortAudioError(msg)
-    return err
-
+    if err >= 0:
+        return err
+
+    errormsg = _ffi.string(_lib.Pa_GetErrorText(err)).decode()
+    if msg is not None:
+        errormsg = "{0}: {1}".format(msg, errormsg)
+        
+    if err == _lib.paUnanticipatedHostError:
+        # (gh82) We grab the host error info here since this function should
```

Comment:
I think a reader who didn't follow our discussion wouldn't know what the alternative to "here" would even be?